api/src/translations.js

```diff
@@ -29,6 +29,18 @@ const extractLocaleCode = filename => {
   }
 };
 
+const validTranslationsDoc = doc => {
+  if (!doc || doc.type !== DOC_TYPE || !doc.code) {
+    return false;
+  }
+
+  if (_.isObject(doc.generic) || _.isObject(doc.values)) {
+    return true;
+  }
+
+  return false;
```

Comment:
It would be worth logging something before returning `false` so it's obvious why your language isn't available.